examples/src/python/example/README.md

```diff
@@ -394,7 +394,7 @@ Pants runs Python tests with `pytest`. You can pass CLI options to `pytest` with
 you could run:
 
     :::bash
-    $ ./pants test.pytest --options='-k foo' examples/tests/python/example_test/hello/greet
+    $ ./pants test.pytest --options='-k req' examples/tests/python/example_test/hello/greet
```

Comment:
It would be even better if, instead of `req` these two bits of doc matched the third below: `examples.tests.python.example_test.hello.greet.greet`. How about `-k greet` and fix the paragraph above to mention `greet`?

Reply:
Sure, sounds good.